documentation/docs/resources/testnet-releases.md

```diff
@@ -3,9 +3,9 @@
 
 Variable  | Value
 ------------- | -------------
-`axelar-core` version | `v0.9.6`
-`axelarate-community` version | `v0.9.1`
-`tofnd` version | `v0.8.0`
+`axelar-core` version | `v0.10.4`
+`axelarate-community` version | `v0.10.0`
+`tofnd` version | `v0.8.1`
 Ethereum Axelar Gateway contract address | `0xCf36e7E9d83772fF1EeeE31D7e3edF66d805A01e`
```

Comment:
Need to update all the contract addresses.  I guess we can leave it for now and update these in a separate PR.